CONTRIBUTING.md

```diff
@@ -247,7 +247,7 @@ Here are some general guidelines
 ## Adding New Configuration Properties
 
 If you want to add a new configuration value to the module, you must modify the following:
- * In `Restore-GitHubConfiguration`, update `$config` to declare the new property along with
+ * In `Import-GitHubConfiguration`, update `$config` to declare the new property along with
```

Comment:
I think you might have missed my previous PR comment about this....so adding it back here for your attention:

I think it would _also_ be great to include a minor documentation update to point out this specific configuration.  It probably makes sense to call it out within the `Configuration` section of the README.

Reply:
I did miss that, my apologies. Still getting used to the GitHub PR interface, I'm more used to the more compact Azure Repos comment boxes and I think I got the two comments mixed up.

Reply:
No worries.  Thanks for adding it, as well as the updates to contrbuting re: keeping those parameters alphabetical.